frontend/src/pages/Player/components/EventStreamV2/EventStreamV2.tsx

```diff
@@ -158,6 +163,7 @@ const EventStreamV2 = function () {
 							data={filteredEvents}
 							totalCount={filteredEvents.length}
```

Comment:
awesome work here! could we also preserve the search state position when there is a event search term applied?
https://www.loom.com/share/50c911ee5236414eb478844dcc3f17c4